oauth_adapter.py

```diff
@@ -35,4 +39,62 @@ def refresh_access_token(self, refresh_token_str):
         """
         token_dict = {'refresh_token': refresh_token_str}
         oauth = OAuth2Session(self.client_id, token=token_dict)
-        return oauth.refresh_token(self.token_url, auth=self.basic_auth)
+        return oauth.refresh_token(self._get_token_info_url(), auth=self.basic_auth)
+
+    def revoke_refresh_token(self, refresh_token):
```

Comment:
moved revoke_refresh_token because it is not specific to fence but part of oauth